src/applications/static-pages/sass/static-pages.scss

```diff
@@ -127,5 +134,20 @@ $formation-image-path: "~@department-of-veterans-affairs/formation/assets/img";
     &.va-top-button-transition-reset {
       transform: translateY(0px);
     }
+    i {
+      font-size: 24px;
+    }
+    @include media-maxwidth($small-screen) {
+      border-bottom-right-radius: 0;
+      border-top-right-radius: 0;
+    }
+    @include media-maxwidth($small-desktop-screen) {
+      align-items: center;
+      display: flex;
+      height: 44px;
+      justify-content: center;
+      padding: 0;
+      width: 44px;
+    }
   }
```

Comment:
Could we move the top button's style into its own module file so that it has its own jurisdiction? Somewhere in `static-pages/sass/modules`

I'm also wondering if can use the mobile-first pattern here instead of `media-maxwidth`? Would mobile-first be a big hassle?

Reply:
I was wondering about extracting the code out of the _entry_ file and i'm glad you said something. 

As far as the mobile first, we can do that. It's just rewriting the same properties that is on the `.usa-button` class to the custom sass styling here. I went with Max Width media queries because it saved a little time and code duplication.

Reply:
If t's a big pain and leads to dup code then this is probably fine. We have just seen weird UI bugs sometimes from when certain elements have breakpoints at width "x" and others at "x + 1px" and it's hard to tell when that could happen in media queries.

Reply:
Yeah, it's only a few extra properties really. So not a big deal. Sticking with the repo standard is probably best. Thanks man.